### PR TITLE
[MRG] Remove impedances in ant reader in favor of an example using antio

### PR DESCRIPTION
**Pull request by mscheltienne**
As briefly discussed here #12861
We could remove the `raw._impedances` non-standard attribute from `ant` and other readers (e.g. `bv`) in favor of an example which showcases how to get this information if needed.

### CODE REVIEW

File: examples/io/read_impedances_from_ant_neuro.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+:class:`~mne.Annotations` with the description set in the argument
+``impedance_annotation``. However, it doesn't extract the impedance values themselves.
+To do so, use the function ``antio.parser.read_triggers``.
+"""
+
+from antio import read_cnt
+from antio.parser import read_triggers
+from matplotlib import pyplot as plt
+
+from mne.datasets import testing
```
Comment (mscheltienne) [inline examples/io/read_impedances_from_ant_neuro.py:20]: I saw another example using the testing dataset, is that OK or should I move this file elsewhere to run the example?

Comment (larsoner) [reply examples/io/read_impedances_from_ant_neuro.py:24]: Yes it's okay to use `testing` as a dataset in examples/tutorials :+1:

**Comment by mscheltienne** [comment]
Render: https://output.circle-artifacts.com/output/job/803d5327-9767-47bf-8cb6-d8ca837c4c01/artifacts/0/html/auto_examples/io/read_impedances_from_ant_neuro.html#sphx-glr-auto-examples-io-read-impedances-from-ant-neuro-py

File: examples/io/read_impedances_from_ant_neuro.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+"""
+.. _ex-io-ant-impedances:
+
+======================================
+Getting impedances from ANT Neuro .cnt
```
Comment (larsoner) [inline examples/io/read_impedances_from_ant_neuro.py:5]: For future compat and not needing to add redirects etc. could we rename this "Getting impedances from different EEG formats" or something, and then have a section with antio, then a short section for BrainVision that trivially shows `raw.impedances`? The latter would really just be a placeholder for when we soon show how to do it with `pybv`.

File: pyproject.toml [hunk 3]
```diff
@@ -251,13 +251,17 @@ ignore-decorators = [
 ]
 "examples/*/*.py" = [
     "D205", # 1 blank line required between summary line and description
+    "D400", # First line should end with a period
```
Comment (mscheltienne) [inline pyproject.toml:254]: I added and exception for D400 in tutorials and examples, as the first line is usually something like:

```
"""
.. _ex-sensitivity-maps:

================================================
Display sensitivity maps for EEG and MEG sensors
================================================
[...]
```

**Comment by mscheltienne** [comment]
Render: https://output.circle-artifacts.com/output/job/42796c97-eeff-433a-b306-9a4e39ae73ba/artifacts/0/html/auto_examples/io/read_impedances.html#sphx-glr-auto-examples-io-read-impedances-py

**Comment by larsoner** [comment]
Thanks @mscheltienne !

**Pull request merged**